README.md

```diff
@@ -900,6 +900,8 @@ $ detox test
 
 ### Troubleshooting
 
+If you get an error like this `Can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)` after launching `bundle install` you can fix it launching this `gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)`
```

Comment:
Can you add a subtitle to the section you added? 
For example: 
```suggestion
### Bundler install error
If you get an error like this `Can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)` after launching `bundle install` you can fix it launching this `gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)`
```

Reply:
done